os/services/shell/shell.c

```diff
@@ -106,7 +106,7 @@ shell_output_lladdr(shell_output_func output, const linkaddr_t *lladdr)
 static void
 output_prompt(shell_output_func output)
 {
-  SHELL_OUTPUT(output, "#");
+  SHELL_OUTPUT(output, "\n#");
```

Comment:
OK. Is this for the case of empty input?

Reply:
This just ALWAYS starts a prompt on newline, in case the previous output was half-baked, etc.

Reply:
I tried this newline thing and I'm not a fan: doesn't help with the case of constantly coming log output anyway, and I IMO the empty lines takes away part of the shell look-and-feel